src/Lykke.Job.TransactionHandler.AzureRepositories/BitCoin/WalletCredentialsRepository.cs

```diff
@@ -60,5 +62,16 @@ public WalletCredentialsRepository(INoSQLTableStorage<WalletCredentialsEntity> t
 
             return string.IsNullOrEmpty(entity.MultiSig) ? null : entity;
         }
+
+        public async Task<IEnumerable<IWalletCredentials>> GetWalletsAsync(IEnumerable<string> clientIds)
+        {
+            var partitionKey = WalletCredentialsEntity.ByClientId.GeneratePartitionKey();
+
+            return await _tableStorage.GetDataAsync(
+                partitionKey,
+                clientIds.Select(WalletCredentialsEntity.ByClientId.GenerateRowKey),
+                100,
```

Comment:
A magic number

Reply:
it's a default value for this method.

Reply:
It is really not obvious when I do review in browser.

Reply:
Agreed. Fixed.